test/end-to-end/content-transform.spec.js

```diff
@@ -86,7 +86,7 @@ describe('apply content transformation', function () {
       .expect(assert.plans.actions.apply())
       .end(async function () {
         const res = await getEntries(SOURCE_TEST_SPACE, environmentId, 'newsArticle');
-        const entriesWithoutSys = res.items.map(i => _.omit(i, 'sys'));
+        const entriesWithoutSysAndMetadata = res.items.map(i => _.omit(i, 'sys')).map(i => _.omit(i, 'metadata'));
```

Comment:
I think you can just do `_.omit(i, ['sys', 'metadata'])`

Reply:
Thanks, I updated this.